src/app/containers/Home.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
+import Aux from '../hoc/Aux';
 
 class Home extends Component {
   render () {
     return (
-      <div>
-        <h5>Authors Haven</h5>
-      </div>
+      <Aux>
```

Comment:
I'm curious as to why you wrapped this in a hoc component

Reply:
If you have one element you can just return that.
In this case, you can return the h5 element. A hoc can be used when you have elements that are on the same level (i.e. not nested).

e.g.
```
<Aux>
    <Navbar/>
    <Home/>
    <Footer/>
</Aux>
```

Reply:
Thanks for giving clarity about this @njeri-ngigi. I had been looking it up. Good work on the file structure Nelson. I am sure after you incorporate the few changes requested by our TTL we shall be able to rebase off develop and adopt this.

Reply:
I was looking at that we might have elements that are on the same level in the nearby future.

Reply:
I understand that. What I'm pointing out is how you used it here, it's not necessary here, you can just return the h5 element as it is